web/src/components/AttributeRow/AttributeRow.tsx

```diff
@@ -1,31 +1,46 @@
-import {useCallback} from 'react';
-import useHover from '../../hooks/useHover';
-import {TSpanFlatAttribute} from '../../types/Span.types';
+import AttributeValue from 'components/AttributeValue';
+import useHover from 'hooks/useHover';
+import {TSpanFlatAttribute} from 'types/Span.types';
+import {IResult} from 'components/SpanDetail/SpanDetail';
+import AttributeCheck from './AttributeCheck';
 import * as S from './AttributeRow.styled';
-import AttributeValue from '../AttributeValue';
 
-interface IAttributeRowProps {
+interface IProps {
+  assertionsFailed?: IResult[];
+  assertionsPassed?: IResult[];
   attribute: TSpanFlatAttribute;
+  onCopy(value: string): void;
   onCreateAssertion(attribute: TSpanFlatAttribute): void;
 }
 
-const AttributeRow: React.FC<IAttributeRowProps> = ({attribute: {key, value}, attribute, onCreateAssertion}) => {
+const AttributeRow = ({
+  assertionsFailed,
+  assertionsPassed,
+  attribute: {key, value},
+  attribute,
+  onCopy,
+  onCreateAssertion,
+}: IProps) => {
   const {isHovering, onMouseEnter, onMouseLeave} = useHover();
-
-  const onCopy = useCallback(() => {
-    navigator.clipboard.writeText(value);
-  }, [value]);
+  const passedCount = assertionsPassed?.length ?? 0;
+  const failedCount = assertionsFailed?.length ?? 0;
 
   return (
     <S.AttributeRow onMouseEnter={onMouseEnter} onMouseLeave={onMouseLeave}>
       <S.TextContainer>
         <S.Text type="secondary">{key}</S.Text>
       </S.TextContainer>
-      <AttributeValue value={value} />
+
+      <S.AttributeValueRow>
+        <AttributeValue value={value} />
+        {passedCount > 0 && <AttributeCheck items={assertionsPassed!} type="success" />}
```

Comment:
Isn't this the same as !passedCount?

Reply:
It's the same as `!!passedCount` but we have a `no-implicit-coercion` lint rule. I could do `Boolean(passedCount)`, but in that case I prefer to do `passedCount > 0`

Reply:
I don't like that rule at all haha